apps/happy-blocks/src/pricing-plans/components/pricing-plan-detail.tsx

```diff
@@ -26,16 +26,17 @@ const PricingPlanDetail: FunctionComponent< BlockSaveProps< BlockAttributes > &
 		} );
 	};
 
+	const CtaLink = attributes.domain
+		? `https://wordpress.com/checkout/${ attributes.domain }/${ plan.pathSlug }`
```

Comment:
Is this link ok or do we need to split it like the other one due to the minification?